corehq/apps/callcenter/app_parser.py

```diff
@@ -49,6 +53,24 @@ def get_indicators_used_in_app(app):
     return indicators
 
 
+def _get_indicators_used_in_forms(app):
+    indicators = set()
+    for form in app.get_forms(bare=True):
+        wrapped_form = form.wrapped_xform()
+        if not wrapped_form.exists():
+            continue
+
+        instance_node = wrapped_form.model_node.find('{f}instance[@src="jr://fixture/indicators:call-center"]')
+        if instance_node.exists():
+            instance_id = instance_node.attrib['id']
+            search_term = "instance('{}')".format(instance_id)
+            for node in wrapped_form.model_node.xpath('*[contains(@calculate, "{}")]'.format(search_term)):
```

Comment:
not 100% sure i'm reading this right, but is this making the assumption that these only appear in calculates? (seems probably ok, though i can imagine some rare/advanced forms maybe using it elsewhere)